libcaf_test/caf/test/fixture/deterministic.cpp

```diff
@@ -13,6 +15,88 @@
 #include "caf/detail/test_export.hpp"
 #include "caf/scheduled_actor.hpp"
 
+#include <string>
+#include <utility>
+
+namespace {
+
+using namespace caf;
+using caf::test::reporter;
```

Comment:
```suggestion
namespace caf::test::fixture {

namespace {
```

(and then drop the opening namespace for `caf::test::fixture` later)